doc/2_loading.md

```diff
@@ -283,12 +282,22 @@ Cassandra Tables which have partitioned with different partition keys.
 ####Join with a generic RDD after repartitioning
 ```scala
 val oddIds = sc.parallelize(0 to 5).filter(_%2==1).map(CustomerID(_))
-val localQueryRDD = oddIds.repartitionByCassandraReplica("test","customer_info").joinWithCassandra("test","customer_info")
+val localQueryRDD = oddIds.repartitionByCassandraReplica("test","customer_info").joinWithCassandraTable("test","customer_info")
 repartitionRDD.collect.foreach(println)
 //(CustomerID(1),CassandraRow{cust_id: 1, address: East Coast, name: Helena})
 //(CustomerID(3),CassandraRow{cust_id: 3, address: Poland, name: Jacek})
 ```
 
+####Preserve partitoner after joinWithCassandraRDD
+
+If you want to keep the previous rdd use the parameter preservePartitioner
+
+In the following example the repartitioning by cassandra replica is preserved
```

Comment:
The "partitioner from reparitionByCassandraReplica is preserverd" Note that the partitioner is not guaranteed to be able to work with the subsequent keys.